lib/checks/color/link-in-text-block-evaluate.js

```diff
@@ -6,6 +6,8 @@ import {
   incompleteData
 } from '../../commons/color';
 
+import { distinguishableLinkEvaluate } from '../../../../a11y-engine-core/lib/commons/distinguishable-link';
```

Comment:
It would be better to have a common import file for importing a11y-engine functions.
Use that common file instead.
Relative path makes thing unreadable and also changes of accidental error increases.